recipes/watchdog-wms/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "watchdog-wms" %}
+{% set version = "1.2.3b" %}
+
+about:
+    home: https://www.bio.ifi.lmu.de/watchdog https://github.com/klugem/watchdog
+    license: GNU General Public License, Version 3
+    summary: 
+       Watchdog, a WMS for the automated and distributed analysis of large-scale experimental data. 
+       The software is implemented in Java and is thus platform-independent. 
+       Main feature include
+        - straightforward processing of replicate data
+        - support for distributed computer systems
+        - customizable error detection
+        - manual intervention into workflow execution
+        - GUI for workflow construction using pre-defined modules
+        - a helper script for creating new module definitions
+        - no restriction to specific programming languages
+        - provides a flexible plugin system for extending without modifying the original sources
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+build:
+  number: 0
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://github.com/klugem/watchdog/archive/v{{ version }}.tar.gz
+    sha256: 7ec16713fcf58dad0bb474e2fe7ded3064d86e48e9baf2408ec9a69be5e580db
+
+requirements:
+  build:
+      - openjdk >=8
+      - coreutils >=8
+  run:
+      - openjdk >=8
+      - coreutils >=8
+
+test:
+  files:
+      - workflow.test.xml
```

Comment:
This is not a valid test case. Sorry for being late on this. Can you please add a test case that runs the software. It is sufficient to call it with --help or so. We just need to see if it executes. Also, the `files` is not supported by our testing framework.

Reply:
There is a test script (run_test.sh) that calls the binary with
watchdog-cmd --help 
and 
watchdog-cmd -x "workflow.test.xml".

Reply:
Is there an additional testing framework apart from the travis CI tests ?

Reply:
can you add `watchdog-cmd --help` and Co just as test/commands and add all other more elaborated test directly into the build.sh script for the moment.

I think this would help a lot.
Thanks.

Reply:
Sure, I can do that. Can you explain me what's wrong with the run_test.sh file I wrote ?

Reply:
There is nothing "wrong" with it per se. Its just that we are also testing your recipes in a busybox based Docker container to make sure you have defined all requirements. In this Docker container we don't have Conda and not test-environment. Which makes it hard to test run_test.sh files.

Reply:
Thanks for the explanation. I will make the proposed changes.

Reply:
The changes are made are in #7001. Any comments on this one ? :)